tools/commands-packages-query.js

```diff
@@ -62,6 +63,44 @@ var formatDependencies = function (versionDependencies) {
   return dependencies;
 };
 
+// Convert package exports into a more helpful format for output in
+// 'meteor show'.
+var formatExports = function (rawexports) {
+  var exports = _.map(rawexports, function (exp) {
+    var arches = exp.architectures;
+    // Replace 'os' (what we store) with 'server' (what you would put in a
+    // package.js file). That's more user friendly, and avoids confusing this
+    // with different OS arches used in binary packages.
+    if ( _.indexOf(arches, "os") !== -1) {
+      arches = _.without(arches, "os");
+      arches.push("server");
+    }
+    // Sort architectures alphabetically.
+    arches.sort();
+    return { name: exp.name, architectures: arches };
+  });
+  // Sort exports alphabetically by name.
+  return _.sortBy(exports, "name");
+};
+
+// Convert package exports into a pretty, Console non-wrappable string.
+//
+// Goes through the package exports. If an export is only declared for certain
+// architectures, mentions those architectures in a user-friendly format.
```

Comment:
All right, I originally thought that the class suggestion was overengineering. But now I am kind of more convinced.